src/sentry/models/group.py

```diff
@@ -705,3 +707,12 @@ def issue_type(self):
     @property
     def issue_category(self):
         return GroupCategory(self.issue_type.category)
+
+
+@receiver(pre_save, sender=Group, dispatch_uid="pre_save_group_default_substatus", weak=False)
+def pre_save_group_default_substatus(instance, sender, *args, **kwargs):
+    if instance:
+        if instance.status == GroupStatus.UNRESOLVED and instance.substatus is None:
+            instance.substatus = GroupSubStatus.ONGOING
+        if instance.status == GroupStatus.IGNORED and instance.substatus is None:
+            instance.substatus = GroupSubStatus.UNTIL_ESCALATING
```

Comment:
LGTM since we do want to default to `until_escalating`. Just need to make sure we check the feature flag before actually handling the substatus

Reply:
You're referring to `organizations:escalating-issues` this guy here right? Tried scanning the codebase for this usage and maybe my local is out of sync but don't see any references to it. 

We don't need to have it set for writing the `substatus` column values, right? Just wanna feature flag the usage/read of that value?

Reply:
Yup, it's a new flag - we haven't added any logic that's flag-gated yet. 

> We don't need to have it set for writing the substatus column values, right? Just wanna feature flag the usage/read of that value?

Yup! Mostly leaving that comment here so anyone looking at the PR in the future is aware